recipes-debian/libpam-ssh/libpam-ssh_debian.bb

```diff
@@ -0,0 +1,43 @@
+SUMMARY = "This pluggable authentication module (PAM) provides single sign-on using secure shell (SSH) keys."
```

Comment:
Please add ``` REQUIRED_DISTRO_FEATURES = "pam" ``` to describe this recipe needs pam feature.

Reply:
Thank you. I fixed it.